### DISCUSSION TITLE
Manually added Basemodell or LORA is not listed, after manually copied to the right folder

### DISCUSSION THREAD

**Discussion by psychonaut97** [lllyasviel/Fooocus#1209] [Q&A]
Hello, i wanted to add some Basemodells / loras. I copied them to the right folder

"A:\Fooocus Version 2-1-791\Fooocus\models\checkpoints"

its: for example: "realisticStockPhoto_v10"
but after i added them, i cant select them in the browser (i restared everything twice) and also tried to click "refresh all files" but still not shown up.

Why?  Can anyone help me out?

**Comment by AgentFUD** [comment]
@psychonaut97  You have to copy it to **Fooocus/models/loras** subfolder.

**Comment by Phrixus2023** [comment]
click refresh
